haproxy.cfg

```diff
@@ -27,12 +27,14 @@ frontend http_front
     mode http
     log global
 
+    redirect scheme https code 301 if !{ ssl_fc }
+
+frontend https_front
```

Comment:
`git` got turned around here as I pulled content from the HTTP service into the `HTTPS` service. Look for ♻️ icon to indicate a clarifying comment.